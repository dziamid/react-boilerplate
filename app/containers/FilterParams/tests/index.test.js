// import FilterParams from '../index';

import expect from 'expect';
// import { shallow } from 'enzyme';
// import React from 'react';

describe('<FilterParams />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
