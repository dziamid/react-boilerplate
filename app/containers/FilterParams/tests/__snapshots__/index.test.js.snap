exports[`<FilterParams /> should render with default props 1`] = `
<div>
  <div>
    <div>
      <Field
        component={[Function]}
        dataSource={
          Array [
            Object {
              "text": "IT and Computers",
              "value": "5836b54d073234521c49c874",
            },
          ]
        }
        fullWidth={true}
        label={
          <FormattedMessage
            defaultMessage="Job Category"
            id="app.containers.FilterParams.selectCategory"
            tagName="span"
            values={Object {}} />
        }
        name="category"
        noFreetext={true}
        onNewRequest={[Function]}
        openOnFocus={true} />
    </div>
    <div>
      <Field
        component={[Function]}
        dataSource={Array []}
        fullWidth={true}
        label={
          <FormattedMessage
            defaultMessage="Job Sub Category"
            id="app.containers.FilterParams.selectSubCategory"
            tagName="span"
            values={Object {}} />
        }
        name="subCategory"
        noFreetext={true}
        onNewRequest={[Function]}
        openOnFocus={true} />
    </div>
    <div>
      <Field
        component={[Function]}
        label={
          <FormattedMessage
            defaultMessage="Filter Results"
            id="app.containers.FilterParams.filter"
            tagName="span"
            values={Object {}} />
        }
        name="title" />
    </div>
  </div>
</div>
`;
